uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveRequestTest.php

```diff
@@ -698,7 +698,7 @@ public function testCalculateBalanceChangeForALeaveRequestForAContactWithMultipl
    * @expectedExceptionMessage Leave Requests should have a start date
    */
   public function testALeaveRequestShouldNotBeCreatedWithoutAStartDate() {
-    LeaveRequest::create([
+    LeaveRequest::validateParams([
```

Comment:
Overall comment about the changes on this file: The test names do not match what it's being tested anymore. The test names mention the creation of a LeaveRequest, but now we're only testing the validation, so the names have to be updated. Also, we need at least one test to check that validateParam() will be called inside the create() method.